cfme/fixtures/cli.py

```diff
@@ -64,8 +73,8 @@ def dedicated_db_appliance(app_creds, unconfigured_appliance):
     '1' selects internal db, '1' use partition, 'y' create dedicated db, 'pwd'
```

Comment:
Please format the docstring .
Example :  https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/cli/test_appliance_console.py#L59